File: fstmerge/examples/Stack_python/Stack_pythonMerge/stack.py
class  Stack :
	~~FSTMerge~~ items = llist() ##FSTMerge## items = list() ##FSTMerge## items = xlist()
	   
	def push(self, item):

		self.items.append(item)

		
	def pop(self):

		if len(self.items) > 0:

			return self.items.pop()

		else:

			return None

	def flush(self):

		pass


	serialVersionUID = 42

File: fstmerge/examples/Stack/Serializable/Stack.java
import java.util.LinkedList;
import java.io.Serializable;

public class Stack<T> implements Serializable {
	private static final long serialVersionUID = 42;
	public LinkedList<T> items = new LinkedList<T>();
	public void push(T item) {
		items.addFirst(item);
	}
	public T pop() {
		if(items.size() > 0)
			return items.removeFirst();
		else
			return null;
	}
}